data/src/main/java/me/androidbox/data/worker_manager/SyncAgendaItemsImp.kt

```diff
@@ -0,0 +1,60 @@
+package me.androidbox.data.worker_manager
+
+import androidx.work.*
+import me.androidbox.data.local.dao.EventDao
+import me.androidbox.data.local.dao.ReminderDao
+import me.androidbox.data.local.dao.TaskDao
+import me.androidbox.domain.constant.SyncAgendaType
+import me.androidbox.domain.work_manager.SyncAgendaItems
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+class SyncAgendaItemsImp @Inject constructor(
+    private val workManager: WorkManager,
+    private val eventDao: EventDao,
+    private val taskDao: TaskDao,
+    private val reminderDao: ReminderDao
```

Comment:
I am passing these as parameters and then fetching the ids and then passing them as `workDateOf` to the workmanager. But would it be better to just pass these  `eventDao`s directly in the constructor of the `SyncAgendaItemsWorker()` Which is better?